hardware_interface/include/hardware_interface/robot_hardware.hpp

```diff
@@ -18,10 +18,13 @@
 #include <string>
 #include <vector>
 
+#include "hardware_interface/actuator.hpp"
```

Comment:
so you would change RobotHardware inplace? is it the goal to change it completely with system?

Maybe to add this changes even one level-deeper in `RobotHardwareInterface`?

Reply:
The robot hardware interface is really only an interface with abstract functions for each implementation to adhere.